### ISSUE TITLE
GUI Shows TV URL Not Name

### ISSUE THREAD

**Issue by Geyup** [PopcornTimeCE/desktop#36]
In Linux the GUI shows in the upper left when you mouse over TV Series, you get an URL not a name.

See screen shot;

![popcorn](https://cloud.githubusercontent.com/assets/13072581/11734280/8fb6bf5c-9fad-11e5-8149-bdadc44e7a23.jpg)

**Comment by Breaking1** [comment]
That's how it's suppose to be, when you hoover the mouse over tv show section you can see which api you are connected too, that way you don't how to check settings to know which api you are connected to. 

Same when you are on movie section, it will only show which api you are using when you are in the tv show section, so until you enter tv show section it will only have the msg  "TV API Not Selected " if you hover the mouse over it.

**Comment by Draxler** [comment]
@Breaking1

No. You are wrong on that. That's never was the case. Popcorn Time always only use the name of the source, not complete URL of the source when hover over Movie or TV Series section.

![33](https://cloud.githubusercontent.com/assets/7681671/11767315/d3ed1858-a1a9-11e5-914d-6bdf8873f9ad.PNG)

**Comment by Breaking1** [comment]
Yeah, but it got changed for the CE version. So no i'm not wrong.

**Comment by Draxler** [comment]
@Breaking1

It doesn't matter who actually changed this or if it's already changed for CE version. 
This change is wrong.
Video sections (Movies, TV Series & Anime) always have only Source Name, not Source URLs (for a good reason). Not only, it's not consistent with the other video sections, Movies and Anime who only uses source names, it's also conflicts with the design with this much info on hover..
It's over the top info which doesn't serve any purpose up there to a user. Settings are for such detail info, where user should go and checked which API is in use atm.  I also think displaying (promoting) source API URLs on ''every corner'' so to speak, is not  good idea to begin with, for some other reasons. These URLs should changed privately (behind the scenes) and also check such info privately - Settings. Imo, P.T. should only use TVApi name (as always) at all times on hover for TV Series section, no matter what API changes are happening in the background. 
  
If this won't be changed back , then at least users should have an option in the Setting to have ability to turned off  ''hover info'' over video sections; Movies, TV series and Anime. This would be right thing to do, to at least give us users this option. 

Regards

**Comment by Breaking1** [comment]
I can't do anything about what you think is wrong, but in regarding your statement i'm right. What was the case for the .io version doesn't mean it will be the case regarding the CE version. That's the beauty about different forks from the same source code, they can all do what they want with it.

**Comment by Draxler** [comment]
@Breaking1 

i'm not arguing that because something was on .io that now ti should be on CE . Do yo even read what i'm saying where the problem is ? Or is it an ego thing or what ?

And i'm also concerned over your last statement (i hope you speak only in your name).
I hope this is community driven project and users could raised concerns if they have some, without being brush of like on some private project , i hope this is not the case. I hope this is not some private driven project with ONLY one sided vision without community being involved. Community usually support only community driven project not private project where no one could say anything about the project. This is how it always worked with Popcorn Time community, we were always able to work with OUR developers in terms of ideas, suggestions, or concerns .. There are a lot of ideas from regular users which were integrated into Popcorn Time app over the time. This is how things worked and you should know this.. BTW, this project is raised from .io community and also it present it self as community driven project who represent .io community, i imagined this is indeed the case..

And If you still think that i raised my primary issue only because something was done in .io version this way and in CE version in some other way, then you should stop replying to me, because you obviously didn't/don't read what i wrote/write for a second time now and you only concentrate on one thing; who's wrong and who's right, which tells me, it's an ego thing for you and not what is best for the project.

**Comment by Breaking1** [comment]
Just came down, i don't claim to be any PT god or anything. Everyone is entitled to his/her opinion.

I just tried to explain why it was behaving like that, so you could see that it wasn't a bug/issue but something the dev's changed.

I don't come here to argue with people.

**Comment by Draxler** [comment]
@Breaking1

I'm only saying that imo this is not right the way it is. I also explained why is not OK.  This was issue report where someone thought it's a bug but never the less, if he wouldn't be bothered with that (change), he wouldn't reported. I just replayed to all that where i'm saying that me to thinks it's no OK the way it is now.
It's also true i should choose deferent words in my first post for my first text i used before my screen-shot. You correctly replayed to a user but i use wrong language in my replay. 
I should say ''This (change) is wrong'' something like that , not ''you are wrong'' . So ,, My bad.

**Comment by Geyup** [comment]
Hi Guys,

Hope not to much heat flying around... LOL

To begin with, I hope we all consider ourselves Geeks enough, that we know what an URL is.

This is not about personal opnions, but guidelines and standards that have been around. Microsoft, Apple, Gnome, and KDE have design standards in place.... These are just a few, I'm sure if we look, we'll find others too...

https://dev.windows.com/en-us/desktop/design
https://developer.apple.com/
https://developer.gnome.org/
https://techbase.kde.org/Development/Guidelines

We also have ```GNU Coding Standards```
http://www.gnu.org/prep/standards/standards.html

Guidelines
https://en.wikipedia.org/wiki/Human_interface_guidelines

A developer can certainly do whatever they please, but when you consider the guidelines and standards that have been in place for many years, seeing an URL on the face of a GUI is not common place. Personally in all the years I've been in computing, across several platforms, Unix, Linux, OSX and Windows, I've never seen this till now.

Now, for my opinion, I don't think it looks clean or professional, and when it was mentioned the URL helps tell us what API we're using. I agree to the point, for what an ```URL``` is!, An ```address``` that tells us the ```address name```, but we don't need to know the address name in this situation, because we are not going to look it up online, we just need to know the name, not an address.

It's pretty clear to see in the settings that this has to do with ```EZTV``` and I believe, just having this name appear in the GUI, pretty much makes it just as easy, and professional looking, to tell us what we're dealing with. Keep the URL in the Preferences and the name on the GUI, I think most developers will agree.

I don't think we need to have a discussion on what an URL is, and it's purpose, and it's origin?

https://en.wikipedia.org/wiki/Uniform_Resource_Locator

Keep up the great work! :)
